test/videojs-contrib-hls.test.js

```diff
@@ -1096,6 +1096,121 @@ QUnit.test('playlist 404 should blacklist media', function(assert) {
   assert.equal(this.player.tech_.hls.stats.bandwidth, 1e10, 'bandwidth set above');
 });
 
+QUnit.test('detect if the playlist stuck at the end', function(assert) {
```

Comment:
Probably something like "detects if we are stuck at the playlist end" or "detects if the player is stuck at the playlist end"